core/domain/question_domain.py

```diff
@@ -459,6 +459,58 @@ def _convert_state_v34_dict_to_v35_dict(cls, question_state_dict):
 
         return question_state_dict
 
+    @classmethod
+    def _convert_state_v35_dict_to_v36_dict(cls, question_state_dict):
```

Comment:
Ditto comments in state_domain here also.

Reply:
Addressed above.